frontend/src/app.tsx

```diff
@@ -1,16 +1,21 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import styles from './app.module.css';
 import { Chat } from './components/chat';
 import { Input } from './components/input';
 import { useMessages } from './useMessages';
+import { ConnectionStatus } from './components/connection-status';
+import { WebsocketContext } from './session/websocket-context';
 
 export const App = () => {
   const { sendMessage, messages, waiting } = useMessages();
 
+  const { isConnected } = useContext(WebsocketContext);
+
   return (
     <div className={styles.container}>
       <Chat messages={messages} waiting={waiting} />
       <Input sendMessage={sendMessage} />
-    </div>
+      <ConnectionStatus isConnected={isConnected} />
```

Comment:
I would quite like to see the dot inside the chat window area, opposed to the top of the screen. What do you think on this?

Reply:
Yeah happy to move it